Sources/Analytics/Analytics.swift

```diff
@@ -4,7 +4,7 @@ import Foundation
 public enum Analytics {
 
     /// A typealias representing a dictionary of analytics parameters.
-    public typealias Parameters<Key: AnalyticsParameterKey> = [Key : Any]
+    public typealias Parameters<Key: AnalyticsParameterKey> = [Key: Any]
```

Comment:
even though this style guide "rule" might change, I though that this was fixed in the latest SwiftLint version? 🤔

Reply:
Apparently not 😞 . I believed it was fixed but it was because I ran `seiftlint autocorrect` which corrected these occurrences, leading me to think that the exception to the rule was working 😞 .

I did revert the dictionaries to have one space around the colons in dictionaries.